pkg/proto/common/v1/common.proto

```diff
@@ -0,0 +1,85 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+syntax = "proto3";
+
+package pkg.proto.common.v1;
+
+import "google/protobuf/any.proto";
+
+option csharp_namespace = "Dapr.Client.Autogen.Grpc.v1";
+option java_outer_classname = "DaprProtos";
+option java_package = "io.dapr.v1";
+option go_package = "github.com/dapr/dapr/pkg/proto/common/v1";
+
+// HTTPExtension includes HTTP verb and querystring
+// when Dapr runtime delivers HTTP content.
+// 
+// For example, when callers calls http invoke api
+// POST http://localhost:3500/v1.0/invoke/<app_id>/method/<method>?query1=value1&query2=value2
+// 
+// Dapr runtime will parse POST as a verb and extract querystring to quersytring map.
+message HTTPExtension {
```

Comment:
It might seem like a good idea to move these to common now, but in future if any field needs to changed/added for dapr internal communication, these common types needs to be changed. OR a new type would be added to handle dapr internal communication to avoid breaking changes. 
I would suggest to keep clear separation between dapr public and dapr intenal proto messages so they can revised independently of each other. This would mean some duplication but would set us up on the right path.

Reply:
That's good. I can make daprinternal to not refer common.proto. This is what I exactly wants to get the feedbacks from you guys. 

All definitions in common.proto will be referred by user-facing dapr and daprclient service definitions.

Internal gRPC service will not use common.proto.

Reply:
> That's good. I can make daprinternal to not refer common.proto. This is what I exactly want to get the feedbacks from you guys.
> 
> All definitions in common.proto will be referred by user-facing dapr and daprclient service definitions.
> 
> Internal gRPC service will not use common.proto.

@amanbha I removed all dependencies to common/v1 in daprinternal. Please review my last change. 

cc/ @yaron2